static/app/views/dashboardsV2/widgetBuilder/header.tsx

```diff
@@ -69,6 +69,12 @@ function Header({
           >
             {t('Give Feedback')}
           </Button>
+          <Button
+            external
+            href="https://docs.sentry.io/product/dashboards/custom-dashboards/#widget-builder"
+          >
+            {t('Read the docs')}
```

Comment:
Add 'Read the docs' button
![image](https://user-images.githubusercontent.com/29228205/154966587-0c758f3f-5f2a-4c9b-96c8-7d98061a0ae9.png)